lib/tasks/db.js

```diff
@@ -53,13 +104,17 @@ module.exports = {
         migrator.pending()
         .then(function (seeders) {
           if (seeders.length === 0) {
-            console.log('No seeders were found.');
+            console.log('No seeders found.');
             process.exit(0);
           }
-        }).then(function () {
-          return migrator.up();
+
+          return migrator.up(_.chain(seeders).pluck('file').value());
```

Comment:
nevermind, I thought you meant something else.. you me to change this

```
{migrations: _.chain(seeders).pluck('file').reverse().value()}
```

to this 

```
_.chain(seeders).pluck('file').reverse().value()
```

or vice versa, as long as both have the same logic